WatchApp Extension/Managers/LoopDataManager.swift

```diff
@@ -100,19 +100,11 @@ extension LoopDataManager {
         }
     }
 
-    func addConfirmedBolus(_ bolus: SetBolusUserInfo) {
-        dispatchPrecondition(condition: .onQueue(.main))
-
-        activeContext?.iob = (activeContext?.iob ?? 0) + bolus.value
```

Comment:
Unlike the active carbs case, this line wasn't harmful, since a bolus is always initiated 'now' and thus its full volume becomes active immediately—but this code is redundant since the values are kept up-to-date.

Reply:
For both iob and cob, what if there is a watch to/from phone communication issue? Let's say the bolus programmed from the watch is successfully sent to the phone, but loses connectivity immediately thereafter and the corresponding `sendWatchContextIfNeeded` that contains the updated iob and cob isn't received. Would the watch displayed iob and cob be incorrect in this case?

Reply:
They'd be incorrect, yes. I'll implement some staleness checking to blank out the displayed values if they're > 15 minutes old (I was thinking they already did, but this prompted me to check!). Accounting for an immediate connectivity loss after a bolus would probably require more complex logic.